engines/bmc.cpp

```diff
@@ -25,6 +25,7 @@ Bmc::Bmc(const Property & p, const TransitionSystem & ts,
          const SmtSolver & solver, PonoOptions opt)
   : super(p, ts, solver, opt)
 {
+  engine_ = Engine::BMC;
```

Comment:
This isn't important at all, but I'm not sure why this diff is showing up. This line is already in `master`: https://github.com/upscale-project/pono/blob/f91ef4d6d8459a219712f1c3d120ed10b8cfd8be/engines/bmc.cpp#L28

Reply:
yeah, i am also curious. I tried to rebase, but it was giving conflicts, so I didn't do it